src/components/Page.tsx

```diff
@@ -133,37 +70,5 @@ export const Page: React.FunctionComponent<{ project?: Project }> = ({
     }
   }
 
-  const project = directProject || storeProject
-
-  return (
-    <PageContainer>
-      {project && (
-        <ViewsBar>
-          <IconBar>
-            <Tip title={'Edit ⌥⌘3'} placement={'right'}>
-              <ViewLink
-                className={!enabled ? 'active' : ''}
-                onClick={() => setEnabled('')}
-              >
-                <StyledEditProjectIcon />
-              </ViewLink>
-            </Tip>
-
-            <Tip title={'Library ⌥⌘4'} placement={'right'}>
-              <ViewLink
-                className={enabled === LIBRARY ? 'active' : ''}
-                onClick={() => setEnabled(LIBRARY)}
-              >
-                <ProjectLibraryIcon />
-              </ViewLink>
-            </Tip>
-          </IconBar>
-
-          <Support />
-        </ViewsBar>
-      )}
-
-      {selectContent(enabled, children)}
-    </PageContainer>
-  )
+  return <PageContainer>{selectContent('', children)}</PageContainer>
```

Comment:
no need for that render children directly

Reply:
please check now